examples/with-react-apollo/README.MD

```diff
@@ -0,0 +1,19 @@
+# Razzle With Apollo Example
+
+## How to use
+Download the example [or clone the whole project](https://github.com/jaredpalmer/razzle.git):
+
+```bash
+curl https://codeload.github.com/jaredpalmer/razzle/tar.gz/master | tar -xz --strip=2 razzle-master/examples/with-react-apollo
+cd basic
+```
+
+Install it and run:
+
+```bash
+yarn install
+yarn start
+```
+
+## Idea behind the example
+This is a basic, bare-bones example of how to use razzle with apollo.
```

Comment:
This could use some more information. Also should have a link to Apollo repo and docs